src/platform/silabs/SiWx917/BUILD.gn

```diff
@@ -77,22 +73,48 @@ static_library("SiWx917") {
     "SiWxPlatformInterface.h",
   ]
 
-  if (chip_enable_ota_requestor) {
-    sources += [
-      "${silabs_platform_dir}/OTAImageProcessorImpl.h",
-      "OTAImageProcessorImpl.cpp",
-    ]
-  }
-
   public_deps = [
     "${chip_root}/src/app/icd/server:icd-server-config",
     "${chip_root}/src/platform:platform_base",
+    "${silabs_platform_dir}/platformAbstraction:wisemcu",
   ]
   deps = [
     "${chip_root}/src/platform/logging:headers",
     "${sl_provision_root}:headers",
   ]
 
+  if (chip_enable_multi_ota_requestor) {
+    if (chip_enable_multi_ota_encryption) {
+      sources += [
+        "${silabs_platform_dir}/multi-ota/OtaTlvEncryptionKey.cpp",
+        "${silabs_platform_dir}/multi-ota/OtaTlvEncryptionKey.h",
+      ]
+    }
+
+    if (chip_enable_ota_custom_tlv_testing) {
```

Comment:
why is this gn arg contain the working `_testing`? It doesn't seem to be relevant to tests only

Reply:
As I understand  with this arg custom processors can be enabled to show how a user would add custom generic file to be "OTA' for upgrade. Its mainly for testing custom processors OTA upgrade and added by Michael.